### PR TITLE
Fix log size for Win build

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

**Comment by rsinukov** [comment]
CI is still failing because of the log size

**Comment by e5l** [comment]
Thanks, will try to reproduce it locally.

File: ktor-client/ktor-client-jetty/jvm/test/io/ktor/client/engine/jetty/JettyHttp2EngineTest.kt [hunk 1]
```diff
@@ -4,17 +4,24 @@
...
 import io.ktor.client.tests.utils.*
 import io.ktor.test.dispatcher.*
 import io.ktor.utils.io.errors.*
+import org.slf4j.*
 import kotlin.test.*
 
 class JettyHttp2EngineTest {
 
+    init {
+        (LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME) as? Logger)?.level = Level.ERROR
```
Comment (rsinukov) [inline ktor-client/ktor-client-jetty/jvm/test/io/ktor/client/engine/jetty/JettyHttp2EngineTest.kt:22]: can you do it through xml config?

File: ktor-client/ktor-client-tests/jvm/test/io/ktor/client/tests/MultithreadedTest.kt [hunk 2]
```diff
@@ -14,9 +14,9 @@ import kotlin.test.*
 private const val TEST_SIZE = 100_000
 private const val DEFAULT_THREADS_COUNT = 32
 
-class MultithreadedTest : ClientLoader() {
+class MultithreadedTest : ClientLoader(timeoutSeconds = 10 * 60) {
     @Test
-    fun numberTest() = clientTests {
+    fun numberTest() = clientTests() {
```
Comment (rsinukov) [inline ktor-client/ktor-client-tests/jvm/test/io/ktor/client/tests/MultithreadedTest.kt:19]: redundant brackets?

**Review by rsinukov — APPROVED** [review]
LGTM

File: ktor-client/ktor-client-tests/jvm/test/io/ktor/client/tests/MultithreadedTest.kt (hunk 2 above)
Comment (e5l) [reply ktor-client/ktor-client-tests/jvm/test/io/ktor/client/tests/MultithreadedTest.kt:19]: Sure, will remove

File: ktor-client/ktor-client-jetty/jvm/test/io/ktor/client/engine/jetty/JettyHttp2EngineTest.kt (hunk 1 above)
Comment (e5l) [reply ktor-client/ktor-client-jetty/jvm/test/io/ktor/client/engine/jetty/JettyHttp2EngineTest.kt:22]: yep, will try

**Pull request merged**